### PR TITLE
SEO-174801 - UWP Multiple-H1

### PR DESCRIPTION
**Pull request by amyprudencekingstone**
Hi @MallikaRK ,
Worked on multiple H1, alt text correction, image name correction,d and description. Kindly review.

Regards,
Amy Kingstone.

### CODE REVIEW

File: uwp/Ribbon/BackStage.md [hunk 1]
```diff
@@ -76,9 +76,9 @@ Ribbon_backstage.Items.Add(Exit_backstagebutton)
 
 {% endtabs %}
 
-![BackStage_img1](BackStage_images/BackStage_img1.jpeg)
+![BackStage commands in UWP Ribbon](backStage_images/uwp-ribbon-backstage-commands.jpeg)
```
Comment (MallikaRK) [inline uwp/Ribbon/BackStage.md:79]: Change image folder name stil have the capital letter.

Comment (MallikaRK) [reply uwp/Ribbon/BackStage.md:79]: Check the changes in all places.

**Review by MallikaRK — CHANGES_REQUESTED** [review]
commented

File: uwp/Ribbon/BackStage.md (hunk 1 above)
Comment (amyprudencekingstone) [reply uwp/Ribbon/BackStage.md]: made  changes

**Review by Aishwarya-Ganesan — APPROVED** [review]
Changes are fine.

**Pull request merged**